linera-service/src/linera/main.rs

```diff
@@ -1447,10 +1447,11 @@ async fn run(options: &ClientOptions) -> anyhow::Result<()> {
                 validators,
                 shards,
                 testing_prng_seed,
-                table_name: _,
                 policy_config,
                 kubernetes: true,
                 binaries,
+                path: _,
+                storage_config_namespace: _,
```

Comment:
(I forgot: Why don't we need a namespace if it's Kubernetes?)

Reply:
I have not used the Kubernetes as it is complex.
My understanding is that Kubernetes has its own setup which will have its namespaces.